File: FASE 1 - Variáveis e Tipos de Dados/Desafio 3/statusDoAluno.java

//GPTando - Mini-Missões - Fase 1 - Missão 3

import java.util.Scanner;

public class statusDoAluno {

   public static void main (String[] args){
    Scanner entrada = new Scanner (System.in);

        System.out.println("Digite sua nota final: ");
        float notaFinal = entrada.nextFloat();

        if (notaFinal <6) {
            System.out.println("Reprovado!");
        }
        else {
            System.out.println("Aprovado!");
        }
    }
}

File: FASE 1 - Variáveis e Tipos de Dados/Desafio 3/statusDoAluno.cs

//GPTando - Mini-Missões - Fase 1 - Missão 3

using System;

class StatusDoAluno {
    static void Main() {
        Console.Write ("Digite sua nota final: ");
        double notaFinal = double.Parse(Console.ReadLine());

        if (notaFinal< 6) {
            Console.Write("Reprovado!");
        }
        else {
            Console.Write("Aprovado!");
        }
    }
}